scripts/PBAB+Collabs/zksync-testnet/contracts/GenArt721CoreV2_zk.sol

```diff
@@ -0,0 +1,705 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+import "./interfaces/0.8.x/IRandomizer.sol";
+import "./interfaces/0.8.x/IGenArt721CoreV2_PBAB.sol";
+
+import "@openzeppelin-4.5/contracts/utils/Strings.sol";
+
+import "@openzeppelin-4.5/contracts/token/ERC721/ERC721.sol";
+
+pragma solidity ^0.8.0;
+
+/**
+ * @title Powered by Art Blocks ERC-721 core contract.
+ * @author Art Blocks Inc.
+ */
+contract GenArt721CoreV2_zk is ERC721, IGenArt721CoreV2_PBAB {
```

Comment:
WDYT about calling this _PBABzk (in case we have _PRTNRzk, etc.)?

Reply:
oh yah good call generally speaking 👍 thought we were abandoning PBAB terminology though, perhaps better to use _ENGINEzk now?

Reply:
SGTM as well 👍